### ISSUE TITLE
[PROPOSAL] Over Powering Scarecrow & Beaver

### ISSUE THREAD

**Issue by adamhannigan** [sunflower-land/sunflower-land#509]
The purpose of this proposal is to explore some of the concerns around the pricing and boost that the beaver and Kuebiko provide.

Some believe it is too powerful and won't be sustainable for the economy. If wood and seeds become free it could negatively affect the overall Sunflower ecosystem.

Before crafting goes live we have the chance to fine tune the behaviour of these items.

**Proposal**

- Kuebiko = Seeds are 75% cheaper
- Foreman Beaver = 1 axe lasts for 5 trees

**Voting**

Please provide feedback on this issue and provide a yay 👍 or nay 👎 reaction.

**Comment by ItsNight** [comment]
way OP

**Comment by adamhannigan** [comment]
@ItsNight What makes you feel this way? Also any suggestions counter-proposals are welcome 🤗

**Comment by Z1kou** [comment]
I definetly Agree with the fact that they are OP, I think the proposed  fixes are  reasonable: What i would suggest is the reduction of ressources of these NFT's since there effect will be still very good but not game breaking, Or going around this a different way would be reducing  the cost for these NFT's by taking into consideration the money and the time taken to gather all the ressources, because what gave them the charm of people wanting to own them is that they are very much OP in terms of farming or wood gathering, So how about keeping the same effects but adding a limit on it?  (ie. every time you sync you have 50%  chance to get all the seeds for free or every 2sync's you get the seeds for free) , Since in the long term the scarcity of SFL and ressources is inevatable so we gotta try to work on balancing the rewards for early adopters and investors and the long term economy !



Basically i am saying :

-Nerf the effects & then price
or
-Keep the effect & the price but limit its effect

**Comment by Labochi** [comment]
I also feel the same way with how the effects of Kuebiko has, it's way to OP with 75% cheaper price for now.

On my opinion, we are still on the early stages of the game and we can add this effect on the other scarecrow as we move forward to the game. I suggest to Nerf it's effect to atleast 50% cheaper but the resources needed to craft Kuebiko will still be the same.

**Comment by AKCH1N** [comment]
I definitely think we should nerf Kuebiko, but on the other hand, I see that the total supply of Kuebiko NFT is only 193 (*from now on*). And there's also the fact that someone who wants to craft a Kuebiko NFT has to go a really long way (*They must craft the first two scarecrows, as well as obtain other items needed for crafting.*)  
First of all, we have to ask ourselves the following question, 

**Would having 200 people have Kuebiko NFTs change the balance of the game?** 
If the answer to this question is yes, I think it would be more accurate to reduce it to 70%. 

**What other solutions can we come up with?**
By increasing the total supply numbers of the Kuebiko NFT and the other two scarecrow NFTs, we can reduce the Kuebiko effect to 50%. Thus, more people can craft Kuebiko and its effects do not change the balance of the game.

**Comment by DonatasDee** [comment]
Honestly I hope it never comes down to simply a vote tally to determine a direction of these. 

A lot of the people don't look a the design decision, but mostly at what the personal power an item will provide.

The reality is that a 200/300 Quantity can almost as OP as one can wish, its quantity negates any effect that it can actually have on the general economy of the game. 200 people getting free wood won't be able to supply the entire community - doesn't matter how anyone might frame it. Same goes for the scarecrow - It still takes time to grow and you can't supply enough of the community to have a real effect.

People need to get in deeper than just a first glance.

**Also, these are more or less FOUNDING NFT's , they're supposed to be more impactful as they are some of the items that only founding members/early adopters can acquire.**

**Comment by DonatasDee** [comment]
Something that I could say though - is that maybe you adjust the beaver after a bigger amount of halvenings that come later on, once the free trees are too good because the SFL needed would really make it a little absurd, that however - could be negated by simply applying the beaver nerf that you have suggested. The scarecrow is not OP because it does nothing for the economy in general. Just make the T3 scarecrow seeds not be tradeable freely.

**Comment by InuBakaBo** [comment]
Problem is people are aiming for certain NFTs and constant changes just puts a bad taste in the mouth as people are spending time and resources for them with the idea that the NFTs have a certain effect. If it's too drastic they would feel like they've wasted all that time and resources as they would probably change their aim to another NFT. I don't know if the effects of a couple of hundred NFTs would effect a 100k player market or if it is OP, need models, maths and statistics to determine if it is.

**Comment by PedrZne** [comment]
For the Foreman Beaver, i think its a really huge nerf. Cause if we checked the skills, at level 10 which says "Axes can be used in 2 trees", there isn't a big difference between this skill and the Foreman Beaver which is very expensive (5k8 wood).

**Comment by SFL-Aeon** [comment]
I agree that the effects are strong yet when you say 200/300 of 100K accounts that’s .2% if you have say old V1 numbers at 300-400k that’s .05%. Plus these comments were mentioned earlier about the effects. It seems weird to all of a sudden change pace when people are a few days away from these. 

Many planned to craft these NFTs for weeks/months. If the cost/effects change for these why not the other nfts that suddenly became more expensive than their counterparts in v1?

I understand nothing is final in beta yet it seems odd to suddenly spring these changes up and leave people out to dry with constant changes on NFTs which is a huge component of the game now.

**Comment by gabrielgcarmona** [comment]
What if we need to feed the beaver to gain its effect during a period of time? We can start with a sunflower and increase it  with another resources after each halvening, so we can adjust if it becomes game breaking. We could just change description for now to something like "Cut trees without axes, but loves a snack".

**Comment by yo121-eth** [comment]
There are indeed two sides to this thing. If the value or capability of NFT does not meet consumer expectations, no one will do it. However, if the price and quantity of the ability remain unchanged, it is too tasteless. This sweet spot is tough to find.

In my opinion, adding a mission mechanism to limit the output of t3nft is a positive reward mechanism that addresses the concerns of new and old beta players from different starting points. Completing the task can also help players obtain t1/2nft and add the conditions of how many tasks or ranch level to meet the production requirements of t3nft.

It can ensure that only hardcore users are eligible to obtain powerful nfts. It can effectively prevent the bubble market value caused by speculative users robbing resources in the market. And increase the owner's esteem for NFT, thereby reducing market liquidity and raising the price of the NFT.

Of course, I strongly recommend that the output of t3nft, such as seeds, wood, etc. Can only be bound to one account for NFT production and daily consumption. Adam does not need to adjust t3nft to prevent them from being too powerful and disrupting the market.

**Comment by lucasfank** [comment]
> Honestly I hope it never comes down to simply a vote tally to determine a direction of these.
> 
> A lot of the people don't look a the design decision, but mostly at what the personal power an item will provide.
> 
> The reality is that a 200/300 Quantity can almost as OP as one can wish, its quantity negates any effect that it can actually have on the general economy of the game. 200 people getting free wood won't be able to supply the entire community - doesn't matter how anyone might frame it. Same goes for the scarecrow - It still takes time to grow and you can't supply enough of the community to have a real effect.
> 
> People need to get in deeper than just a first glance.
> 
> **Also, these are more or less FOUNDING NFT's , they're supposed to be more impactful as they are some of the items that only founding members/early adopters can acquire.**

I fully agree with Dee on this one. Yes, the perks are very good for the individual, however, the price of these items accurately reflect the benefit. In order to craft these NFTs you will have to have farmed the materials for months. And in the general scope of things, a population of 200 to 300 of these NFTs is not going to do any actual harm to the community UNLESS you let the NFTs be traded between accounts and used freely with no cooldown, as this will bypass the time needed to regrow trees and harvest the crops. 

The developers must truly consider the impact 200-300 accounts utilizing these NFTs will have on the market, because it is not as high as it seems with the player base that is playing currently. (I believe the ROI for these NFTs is around 30 days as well).
I propose that the team implement a cooldown on NFT transfers so that they cannot be abused by multi-accounters, or even passed between friends in-between harvests to get free seeds/wood.

**Comment by InuBakaBo** [comment]
as you can see, why would I bother with a kuebiko when I can just multiaccount and get similar results?
(they mentioned you can have 5 farms on one wallet address eventually to specialise different things)

Or even why both spending 600 whole wheat+other resources to upgrade scary scarecrow to kuebiko when I can multiaccount and get better results?
(600 wheat takes (600/22 fields*24  hours= 28 days...))

![image](https://user-images.githubusercontent.com/98318794/162600679-9ae48866-a462-466d-a139-48e38a8087a7.png)

**Comment by agcm19** [comment]
Proposal: set limitations for Kuebiko and Foreman beaver usage 

a newly crafted Kuebiko can only be used for 1000 free seeds newly crafted foreman beaver 50 trees then need to be maintained (enter maintenance/repair shop where  farmers need to supply/provide materials to restore them to their original usage or make them pay a certain amount of sfl for example 100 sfl would restore items to 50% original state 

or

Kuebiko can only be used with a certain seed. The user has to choose which seeds he/she would want to be free of charge forever (like in a big field, 1 scarecrow is not enough to protect all crops I think)

also to encourage players to craft kuebiko and beaver maybe give them other hidden functions (farms with these NFT can either be immuned to attacks/pests) which can be unlocked by crafting/supplying materials or paying certain amount of sfl

**Comment by Alexcnder** [comment]
I think both just need little adjustments

I think the Kuebiko is not OP since in 2 Halvinings it will earn less than a normal Farm now. 

I think they will become OP with the Land expansions thats why i would cap the Kuebiko to just buy a set ammount of seeds a day for free (22 Free Seeds per day)



Foremann Beaver similar to the Kuebiko this thing would be op if the Effect doesnt get fixed till land expansion. I was thinking of changing the Beaver to only be able to Forage the 5 Tress for free. But i would add something like a Halvening to wood aswell (just for the foremann) so when wood reaches point x Farming the 5 Trees with the beaver will give 2.5 Wood instead of 5 wood then 1.25....
My Proposal would be every 5 (just a number i came up with) SFL halvinings the foremann profit halves by that it doesnt lose its value fully

**Comment by lets2i** [comment]
First, we need to look at why we even want NFTs in the first place.
So in my opinion NFTs should be a way to help the player progress further and combat the halvings of the tokens. If the player just sells off their harvested SFL, at some point they wont be able to catch up anymore.

If the player doesn't have good incentive to work towards, in this case this OP NFTs. There's no reason to keep hodl and they will just take profit everyday, until it's not profitable anymore.
Also 200 NFTs aren't even near the number that would really hit the economy. In my opinion they should be OP in the eyes of the player to have them rush towards them and burn resources while doing it.

My view is that tier 1 and 2 are too weak, they should cost more and be stronger to burn even more resources and let the little player, that wont be fast enough to craft the tier3 before it's sold out to have something to works toward.

**Comment by Alexcnder** [comment]
A other approach to Balance the beaver is to make it so add a second feauture to the t2 beaver that allows people for forage 5 Trees for free a day this would combat the fact that supply will come only from 200 people instead from 5000.
The Foreman will keep its OPness with the introduction of land expansion

**Comment by PigaPigaPiga** [comment]
Hello all, I'm a bit new here on the Github SFL Proposal scene, but an avid farmer/community member since early December. I'm here today to tell you all about how these NFTs - in reality - aren't really OP enough, and should potentially either be increased in power or at least adjusted down in price (and perhaps power). 

I have read all the responses so far, and the only thing that actually makes sense/resonates correctly with me are the suggestions made regarding mint supply: I agree, 100-300 NFTs in a game with potentially millions of future users, seem exhaggeratedly scarse. Even if they are "founder" NFTs.. As these are Farming/Gathering enhancing NFTs, naturally their main point of value should be utility over scarcity, and currently it's mostly FOMO fueling the fires. I think they could easily be boosted up to 800-1000 in supply, even for the top models in each Beaver/Scarecrow directions. Why? Because it doesn't make a *huge* difference in the end anyway, it would even make it "more fair" as more people would be able to take part, and way way way more resources would be burned/prolonging the process on the way there. Let's look at the math;

**Free Seeds:** Game breaking? Barely.. Free seeds could be saving a user for around 120 SFL of Seed costs per sync (to empty the shop), which could last 4-5 days (with the speed increases), if the point is to be saving on sync costs. (_Probably isn't_)

In terms of SFL "saved" and potential earnings, @InuBakaBos charts above show that the Kuebiko can earn up to 125 SFL a day planting free potatoes, if it was possible to plant them all day. Compared to the normal profits per day, we get that the Kuebiko helps the Potato-non-stop-planter save up to 99.89 SFL a day in Potato seeds. But of course, nobody can really plant potatoes all day. 
In comparison a normal farmer would also be able to produce 25% of that, without free Seeds. So one Kuebiko player, would only influence the market as much as somebody else who is farming on their 5 alt accounts. Giving them either the luxery of being able to earn up to as much on just 1 account, or to potentially "double" their overall SFL earnings per day.

This sounds good, but is it really? Let's say someone is able to earn 10 SFL a day right now (we can't really), or up to 50 SFL a day playing 5 accounts, in-humanly efficiently: Then it'd take 26 days to earn enough SFL needed, to produce the Kuebiko and on top of that need to farm Wheat for 20 days (on 1 account, or 4 days on 5 accounts). So that's minimum 36 days of hard work dedicated to produce it, during the current halvings rates. 
After that you'll have put over 1300 SFL into your Kuebiko Scarecrow to have unlocked Seeds. If we end up having 100k Beta players soon, from where maybe just 10%, 10k farmers, earn 10 SFL a day, then the next halving would arrive in approximately 45-50 days of farming (not assuming it suddenly grows to 50k-100k active farmers). 
So after having put 36 extremely intensive days of work into producing a Kuebiko, you'd have another 14 days left of the halving. If you continue as a potato planting robot, you'd the fastest be able to have "saved" the 1300 SFL needed to produce the scarecrow in another 26 days... and after the first 14 days there could be a halving, which also efficiently doubles your remaining ROI period from 13-18 days to 26-36 days again .. and before that's over yet another halving might even have happened.

So my point is, you aren't even guarenteed to be able to ROI (and actually start earning profits), even on having free Seeds, if you put 1600 SFL and thousands of hours into producing the Kuebiko. 

Sorry to crush your dreams. Kuebiko could have it's Wheat/SFL craft cost reduced by 33-50%, and it's supply raised x10, and still not be "game breaking" in any way really,  because 69% of players who'd manage to get one, wouldn't be able to use it efficently enough to get it's ROI .. There's the luxury of having 1 account produce up to as much as 5 seperate accounts, and there's the prestige in having one of the very first super strong NFTs. Or there's the added bonus of getting extra sweet time/yield ratios on Radishes, making those viable .. But it isn't gamebreaking.
It's not like it's producing free animals and endless instant Wheat, that'd be gamebreaking, because that loops endlessly ..

And the Kuebiko could just be made not-withdrawable while one holds Seeds. So all seeds must be planted before it can be withdrawn, then people can't get free seeds on alts everywhere forever either.. no problem?




And the same thing goes with the Foreman Beaver, only he costs even more SFL to produce.. A staggering 1600!

And with this guy, there's even two major traps on the way: The first Beaver and the middle Beaver. Those two are gonna be such traps, most might not ROI on too:
First beaver costs approx 101 SFL to craft (=400 Wood worth) and increases Wood drops 20%, to make it worth it you'd have to chop down approximately 2400 wood, from about 600 trees, so 120 rounds of the 5 trees in the Lumberyard! 
With a total cost of 600 SFL in Axes .. So you'd be able to ROI that in like 20-30 days of daily non-stop Wood chopping 

.. before it being paid back .. Earning 600 SFL to get those axes could take you at least 60 days at current halving rates, and we already talked about those .. It could potentailly take at least 3-6 months of very dedicated Wood chopping (which would consume most of farmers+alts SFL prodution along the way)

That's a very very long time to be able to get to ROI on an item.. It of course helps if you get the next tier Beavers, but let's look at the middle Beaver:

Middle Beaver lets you cut Trees twice as fast.. So you just get to push Wood production faster/**spend SFL faster**/"ROI quicker on the 1st Beaver".. Now only 1.5-3 months before the first one is paid back. And as the middle beaver only does those three things, the first beavers ROI mechanic also needs to cover second Beaver, now putting that ones ROI with just the first two beavers at around 3x the first beavers. So 4.5-9 months to earn that ones cost back.

The top beaver, Foreman Beaver, is of course the catch.. "free axes", "the dream".. Is sadly not even actually OP as it sounds, because it nullifies the second Skill Specialization in the lumbering direction: Logger: Axes can be used on 2 trees.
Compared to someone with Logger something everybody could eventually reach, at most Foreman Beaver would save 120 SFL worth of axes a day, if every tree was chopped every hour 24/7. We of course also sleep, so doing it 16hours a day that's 80 SFL worth of extra savings/earnings per day, or about equally to what 8 casual crop farmers would be farming just planting crops -> buying axes normally and farming their trees.

The Foreman Beaver costs around 1600 SFL to produce, so saving 80 SFL a day, you'd be able to earn it back the earliest in 20 days after purchase .. at which point most would also be at risk of not being able to get their ROI back on their NFTs in time. 



Sooo .. **TL:DR**, even at the current "unlimitedly omg omg OP" versions with free seeds and free axes, they might not be directly "OP enough" for 80% of their future holders, because in reality most of those who'll be able to afford them, wont have time to use them 16-20 hours a day to actually reach the point where they'd be earning good "profits" on having crafted it on the other side.

Crafting the NFTs should be a glorious and luxerious and joyous goal, not something to be reached, only to leave you to be even more forced to play intensively to make it worth it.. Yes, they're "a bit OP", but so is having 1200-1600 SFL in general.. In SFF that was worth as much as $10k at one point, of course someone should get an insanely awesome perk by putting potential $10k worth of assets and efforts into an NFT


My "first" / last suggestion, was to bump up the mint supply on those "op" NFTs a bit though.. As these _extreemely_ limited NFTs could potentially steer up more drama and eventually lead more people to drop the game/mechanics in disgust, if they're only limited to these many .. and especially if 5-10% of the supply is just giving away to people who (pardon) "haven't worked for them" .. because yes, I agree in those cases those NFTs are HUUUUUGE rewards and insanely overpowered gifts to be blantantly giving around in multitudes. ;)



-- Piga, monKeyfarmer .. 
Banana tree, wen?

**Comment by PigaPigaPiga** [comment]
Oh, and I forgot: That mostly all only holds true for people who're close to / would be ready to farm them "soon" now around the launch.. These Beaver NFTs would be almost secure to be impossible to ROI, if they were to be made at current costs after the next halving.. So it might only "really" make sense to make them within the next 10-14 days anyway.. after that it'll 100% securely lead to losses, compared to just stacking SFL.. Or adding SFL into the LPs for Wishing Well rewards.

Which doesn't require/demand 16-20 hours of farming every day to make sure to earn back the ROI invested in the NFTs too.*

**Comment by InuBakaBo** [comment]
imo Kuebiko is fine, as it affected by the halving event, whereby foreman beaver should be nerfed a little, but the proposed change absolutely destroys it, especially if the 5000 wood cost is kept.

**Comment by InuBakaBo** [comment]
> imo Kuebiko is fine, as it affected by the halving events and will eventually have dimishing returns, whereby foreman beaver should be nerfed a little, but the proposed change absolutely destroys it, especially if the 5000 wood cost is kept.

**Comment by PigaPigaPiga** [comment]
> > imo Kuebiko is fine, as it affected by the halving events and will eventually have dimishing returns, whereby foreman beaver should be nerfed a little, but the proposed change absolutely destroys it, especially if the 5000 wood cost is kept.

I'd say both are fine to be honest, but yeah. Kuebiko is well balanced in halvings both affect it's efficient daily SFL saving effect gets lower with time and it's costs..

But the Beaver works a bit the same way, that ones cost is also affected by halvings as when fractional axes kick in, the Beaver wouldn't be getting 3-5 Wood per free tree, but only maybe 0.03-0.05 Wood per tree .. and the SFL on axes saved, would also end up being way way less with time .. So all in all the period of ROI and "earnings", for both of them would be somewhat limited and rushed, as one would never really know how soon the next halving could arrive. 

I don't think their costs are too wrong either, but increasing their mints could also help "dampen" their eventual effect too .. As if more people can burn Wood and Wheat to get them, the markets/swap pools for Wood and Wheat will also be more demanded, which will sustain the extra Wood/Wheat these assets will push farmers to aim for and farm.

> >  whereby foreman beaver should be nerfed a little, but the proposed change absolutely destroys it, 

actually the proposed change doesn't do "that much" .. apart from make it less "special" compared to just specializing in the Lumbering route with the Skill tree and picking Lumberjack and Logger, which everybody can do.. 
With Logger an axe works for 2 trees, so having the Beaver work for 5 trees instead increases it's efficiency by 2.5 .. making one Beaver worth about as much as two farms with Logger

In terms of SFL, a free Wood chopping a top Beaver would almost be able to produce 156 SFL worth of Wood a day, at the moment: up to 600 Wood

If we nerf it to last only 5 axes, then it'll cost 48 SFF a day to run in axes, but will still produce 108 SFL worth a day in profit of wood.

So instead of 40 days ROI, we'd have like a 65 day ROI

I agree, that might be more inline with for example Kuebikos cost / profit ratio and ROI .. perhaps, that'd be slightly more 'balanced' in-between the two of them .. but it also makes it even less likely to be able to ROI in time on them. And with Logger also being there, it'd make the whole Beaver-NFT direction nothing but a huge Wood burn dump .. most likely not to be worth it at all

**Comment by lets2i** [comment]
like everybody already mentioned, T3 scarecrow is NOT the issue.
Only thing worth discussing is the Beaver. If Numbers are put in front, then no it's not OP at all, maybe after 2 month, with 3 halvings, but at that point it could be made that trees don't drop 3-5 Woods anymore and also don't require a full axe to chop.

**Comment by PigaPigaPiga** [comment]
A buff of the first two tiers sound like a good idea.

And yeah, fractionalized axes/resources would help balance T3 Beaver long term. However that would also increase the value of especially the 5k Wood needed to produce them now hugely, because Wood would end up getting even harder and harder/more cumbersome to produce in full pieces too..

I don't think it's OP, it costs over 1600 SFL worth of resources to craft, and still needs some ~ 1200 trees, or 480 hours or 20 days of on-cooldown Wood-chopping to produce .. on one farm 

So if it was less OP, it just wouldn't be worth to get it.. It'd just be another Beaver trap for people with too much Wood to fomo into 🧑‍🌾

**Comment by PigaPigaPiga** [comment]
It would perhaps be too OP if the Foreman Beaver chopped all the wood automatically though* 😆

Seeing as it's still something people need to work on/for, most of it's value/usage will either go un-utilized .. It's a super strong NFT, but it's also meant to be that, and I think it's power level matches it's price rather well already compared to the Golden Cauli, T3 Scarecrow and the Coop .. It's all in an OP balance, each worth about 3-8 alt farms worth of daily production, in various fun mechanics and ways .. The way it should be 🧑‍🌾🥳

**Comment by SteveChickenCX** [comment]
Wow, I just read all of your comments and you all are amazing! 
While I do get the argument that 200 mints won't effect the economy that much.

My biggest concern is creating a big gap between Whales and middle-class 

If we call the people who plays by the rules with 5 accounts, build up a strategy and reinvest into the game 80% of the time "Middle-class" 
We don't want to create unattractive markets( like the wood or eggs) market by letting a few people control it.

We should always let middle-class players aboard this markets without having to pay 5k$ but just play for a few months in a smart way.
I kinda compared this game to RPG games where you don't wanna release the strongest item at the start of the game.

RPG games give a player a "Hat" that gives + 5 Strength in the beginning,  later on these  "Hats" scale to 50 Strength.



I propose a scaleable model that will ensure long-term middle-class players a way to keep playing for fun and not losing the current, while giving new players a chance to catch up:


Treat each halving period as a season and work on 4 different NFTs lines:

1. Compounded line : Upgrade to existing lines- add a next tier upgrade like Advanced Scarecrow, but  instead of making it super costy, make it compatible to the season resources gain, the hard part would be to mint the previous “Scarecrows” that would be hard to mint because they were released couple seasons ago and was compatible to an higher resource gain.
This way we reward our dedicated players that chose to craft the previous versions last season. New players would have to craft or buy the previous ones to get the “Advanced” version.


2. Alternate lines- “catch up” lines : still WIP but there needs to be a way for new farmers to catch up on existing features that were released a couple seasons ago. Let’s hold an example:

If we take Chickens , in about 5 halving crafting the chicken coop would be infeasible, new farmers would have to buy a real costy NFT or just leave the Chicken market and focus purely on new features for the current season (buy Parrots or something) 
Hence we have two options to let players catch up on the chicken industry:
* Release a Second version for coop that is suitable to the correct resource gain, make it less powerful but stackable so a player could have two chicken coops.
* Release a new “feature”/ “booster” to chickens that new players would be able to craft ( Chickens can now fight and aid you in battle if you train them, w\e) 

3 .Cosmetic Lines - Cosmetic items that range from high mint to low mint, suitable to the season and with a range of prices, most Cosmetics should be a low cost, and 1-2 should be high cost making them the “Signature” item of the season. 
If we release chickens this seasons we can make a "Golden Statue of Chicken" or a freaking "chicken costume"


4. New Lines- introduce new features to the game and NFT that boost them (fishing, fighting, flying, diggin…) 

small example of the concept

![unknown](https://user-images.githubusercontent.com/99016671/162608588-b22ef0f5-6586-4c52-9ac1-0e597ad8a2bc.png)


### for the matter at hand Ill recommend:

**reduce the cost of Scarecrow and Beaver**

**reduce the effect of Scarecrow and Beaver**

**Increase the mint numbers of Scarecrow and Braver**

Later on work on T4 for them that provide even better benefits but lets start small and build an healthy economy with less tears.

**Comment by SrCouve** [comment]
In my review what is much more detrimental to the game and I have been talking about this for a while **is the multi-account along with the entry fee costing only 1 matic and none SFL** so people thinking about buying NFT will do following calculation "is it worth it if I spend hundreds of matics to buy an NFT or make 3 farms spending 1 matic and get the same gain" then I say and repeat the entry into the game has to be linked to some amount of SFL, in case there is abuse of multi, would at least positively boost the token economy, generating SFL buying volume

**Comment by SrCouve** [comment]
In my review what is much more harmful to the game and I have been talking about this for a while is the multi-account along with the entry fee costing only 1 matic and no SFL, so people thinking of buying NFT will do the same. same . following calculation "it's worth it if I spend hundreds of matics to buy an NFT or make 3 farms spending 1 matic and get the same gain" then I say and repeat the entry in the game has to be linked to some amount of SFL, in that case if there is some abuse of Multi would at least positively boost the token economy, generating SFL purchase volume, only now in Beta, for example, people are abusing the mintage of farms and selling on the high seas, for sure everything is already automated, in the past game this entry value only in matic without any SFL was one of the things that contributed to the abuse of bots, this will always be a problem, people will keep trying to abuse this system in some way, whenever they open a farm for more advantageous than having an NFT, in my opinion this will be harmful to the game economy, a value in SFL would help the token economy absurdly and help reduce multi abuse,

**Comment by tiffanydys** [comment]
You do not need to nerf them right now, maybe in after a few halvenings.

For me adding a note that says (effects are expected to change after a few halvenings) is enough to solve this issue.

**Comment by SFL-Aeon** [comment]
I believe everyone sees free seeds and thinks it’s OP. The math really doesn’t support that on average you spend 11-15SFL on seeds. That’s before the next halving which means it’ll be less.

 If you take into account the costs to grow the wheat (11x23 with t2 = 253) + the loss of unrealized gains (assuming 35 SFL a day so 23x30 = 690) + the mint cost of 300 for a total of almost 1.3K SFL, the t3 scarecrow would take over a year to ROI. 
 
I do agree I don’t like the artificial scarcity. I understand it’s suppose to be founder NFTs or chase NFTs. Give something exciting to cause people to work for or buy. But as it stands it’s really limited to a select audience.


In regards to the beaver it isn’t being regulated by the halving. As wood,iron,stone and gold are based on tool cost and these values never change then our ROI is insane over time. I think beaver chopping trees without axes is still fine but it’s effect could be nerfed. Maybe only a set amount of trees a day.

I don’t like the idea of cumulative upkeep costs. This is just another gimmick that’s not needed. If you do it for these then why not the others?

**Comment by SFL-Aeon** [comment]
> A other approach to Balance the beaver is to make it so add a second feauture to the t2 beaver that allows people for forage 5 Trees for free a day this would combat the fact that supply will come only from 200 people instead from 5000.
> The Foreman will keep its OPness with the introduction of land expansion

I actually like this idea. T2 beaver gets 5 free trees. This goes with chickens idea of increasing gains. I think many would be happy with this change tbh.

**Comment by PedrZne** [comment]
> For the Foreman Beaver, i think its a really huge nerf. Cause if we checked the skills, at level 10 which says "Axes can be used in 2 trees", there isn't a big difference between this skill and the Foreman Beaver which is very expensive (5k8 wood).

Just adding something really quick. If you guys count how much the beaver costs, you will see that its worth to be op like that, i did count it and its approximately 1000 sfl, and 30 days for roi...

**Comment by Laoshi-Ferdy** [comment]
> The purpose of this proposal is to explore some of the concerns around the pricing and boost that the beaver and Kuebiko provide.
> 
> Some believe it is too powerful and won't be sustainable for the economy. If wood and seeds become free it could negatively affect the overall Sunflower ecosystem.
> 
> Before crafting goes live we have the chance to fine tune the behaviour of these items.
> 
> **Proposal**
> 
> * Kuebiko = Seeds are 75% cheaper
> * Foreman Beaver = 1 axe lasts for 5 trees
> 
> **Voting**
> 
> Please provide feedback on this issue and provide a yay 👍 or nay 👎 reaction.



> The purpose of this proposal is to explore some of the concerns around the pricing and boost that the beaver and Kuebiko provide.
> 
> Some believe it is too powerful and won't be sustainable for the economy. If wood and seeds become free it could negatively affect the overall Sunflower ecosystem.
> 
> Before crafting goes live we have the chance to fine tune the behaviour of these items.
> 
> **Proposal**
> 
> * Kuebiko = Seeds are 75% cheaper
> * Foreman Beaver = 1 axe lasts for 5 trees
> 
> **Voting**
> 
> Please provide feedback on this issue and provide a yay 👍 or nay 👎 reaction.

I think the idea of this proposal is really nice. 

By math, we know both of the NFT can be effected by halvening event and some of player maybe will take time to get ROI for the NFT they have minted, but the purpose of this game is play to own right? We have something  in return when we holding an NFT, in the future it's not just only about the money, but the prestige holding one scare NFT will priceless. 

But in real world, there are no free launch. If both of NFT provide something "free" in return while we holding it, I'm afraid it will not sustain in the real world. As we want to create this game similar like real world economy, I think we need to consider to change it like this :

> * Kuebiko = Seeds are 75% cheaper
> * Foreman Beaver = 1 axe lasts for 5 trees

By doing that, it give the NFT holder more benefit than who are not holding it, but it's not way to op and will matching up the purpose of the NFT in this game. 

Good proposal from Adam. Salute from Indonesia

**Comment by Kratka007** [comment]
I totally agree with the proposal to nerf Kuebiko and Beaver.
Especially Kuebiko and I believe that if the reduction was 50% it would still be quite fair.

I believe that 75% is still overpowered, as he accumulates the bonuses of the other 2 scarecrows
Also, it's quite likely that the only people who will be able to craft Kuebiko are those who already have the V1 scarecrow, which was turned into V2.
For those, the benefit they received from starting version 2 with the current scarecrow is HUGE.
I don't remember the scarecrow previously being this strong (in benefits) and in version 1, people paid around 30 dollars to get it.
I agree that whoever made this back then proved to be a great supporter of the game, at that moment of difficulty, but to be honest, I believe that the way it is now, the benefit is well above what would be proportionality

And if that wasn't enough, there is still the possibility of these same people getting Kuebiko more easily (compered to others), which would make them even more OP

**Comment by adamhannigan** [comment]
Some awesome discussion here. Thanks for the detailed responses!

Given support for this proposal is around 60/40% it doesn't feel like a strong enough majority to go forward. Based on this feedback we will try encompass some of the surrounding sentiment here to come up with a more sustainable system.

We will have multiple boosts being released each season and these conversations will help us steer us in the right direction. We won't get the system perfect on the first launch. For certain we will need to see how some of these initial items play out in scarcity + distribution to adjust future items

**Comment by block-kid** [comment]
I'd like to re-stress @lucasfank 's point. These powerful NFTs need a COOLDOWN. Else they will be abused through multi accounting. This is the thing I'm most worried about early on. 

I also agree, long term the Foreman is way OP and needs to be adjusted. A fun mechanic that could fix this is _**overlogging**_, where the number of trees declines over time. E.g. for each halvening one tree disappears permanently, or the time it takes them to grow doubles.

I like the idea of additional seasonal releases of T3 scarecrows. 
- e.g. increasing seed drop rate by 4x
- fractionalised guaranteed seed drops such as minimum 0.25 seeds for every plant harvested( if you don't receive a drop)

**Comment by BanInstinct** [comment]
Totally agree with this, 5 tree lasting ax is much better and cheaper seeds too

**Comment by KKonst27** [comment]
> @ItsNight What makes you feel this way? Also any suggestions counter-proposals are welcome 🤗

![image](https://user-images.githubusercontent.com/103465383/162864726-021dc23d-f8e5-49cf-9b91-28affb652219.png)
https://docs.google.com/spreadsheets/d/1J4M_QrZxcILt5LqADSjYGuLlgU7xJQe6ovGDLN6A2Lw/edit#gid=1736706565

**Comment by Armadi11o** [comment]
Was talking about the scarecrow with a dude in the discord and we reach to an idea to "nerf" it without take out the advantage of to have it. The "free seed" buff may be changed to something like "1~2 times per day the scarecrow gives a pack of any seed" the pack in this case would be the max quantity of the seeds that you can keep of that kind. The "x" times, the CD of this and the kind of seed that would drop may change but this would be a solution (at least for the scarecrow).

**Comment by NoisyPT** [comment]
I think beaver should not be nerfed as it doesn't affect the SFL price.  Also 5000 wood is 1,250 SFL without boosts and should be rewarded for the grind and 300 Beavers in a 100K player base is nothing.
Hope you guys think about the people that already grinded the wood by trading or farming.

**Comment by alivitedg** [comment]
What I buy beaver for $1000 and kuobiko $1300 what you do bro.
you wrote this And since the beginning of the game.
And we bought and paid our money based on this buffs.
Where is the credibility in the game?
You waste our money like this and our calculating
we can't investing more in the future in this game because we are afraid that you will change the buffs and you lose your credibility because there is 500 player invest and lose his money you change the buff.
Are you going to pay $500,000 to buyers of this buff ?
Based on this
Do nft games changes his nft buffs after player buy it and pay his money based in his buff ? 
bro if your nft minted and selled in opensea you don't have permission to change buff 
and in this situation you change buff of expensive nft we don't know this when we buy. 
if we know this we don't buy with this higger price.
bro you don't have permission to do this.
if you force and do this you lose a lot of things the first thing you lose credibility
secondly thing you lose all rich players he buy this and also he buy a lot of sfl and a lot of thinks.
